Formula/apache-drill.rb

```diff
@@ -1,9 +1,9 @@
 class ApacheDrill < Formula
   desc "Schema-free SQL Query Engine for Hadoop, NoSQL and Cloud Storage"
   homepage "https://drill.apache.org"
-  url "https://www.apache.org/dyn/closer.cgi?path=drill/drill-1.14.0/apache-drill-1.14.0.tar.gz"
-  mirror "https://archive.apache.org/dist/drill/drill-1.14.0/apache-drill-1.14.0.tar.gz"
-  sha256 "1145bdbb723119f271d32daf4cdd77cdeebe88ddcb7d04facd585b715bb5723b"
+  url "http://apache.mirrors.ionfish.org/drill/drill-1.15.0/apache-drill-1.15.0.tar.gz"
+  mirror "http://www.gtlib.gatech.edu/pub/apache/drill/drill-1.15.0/apache-drill-1.15.0.tar.gz"
```

Comment:
Please keep the servers as indicated: `www.apache.org/dyn/closer.cgi` as `url`, and `archive.apache.org` as `mirror`.